docs/DeveloperGuide.md

```diff
@@ -827,8 +827,20 @@ testers are expected to do more *exploratory* testing.
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
+   
+1. Shutdown
```

Comment:
probably can improve clarity by changing it to `Quitting iVolunteer` or something

Reply:
oki